DEVELOPMENT.md

```diff
@@ -64,11 +65,13 @@ workloaddefinition.core.oam.dev/deployments.apps                      deployment
 workloaddefinition.core.oam.dev/statefulsets.apps                     statefulsets.apps
 ```
 
-## Test
+## E2E test
 ```
-$ go test ./pkg/test
-ok  	github.com/cloud-native-application/rudrx/pkg/test	14.662s
+$ cd pkg/test
+$ ginkgo -v -r
```

Comment:
put this into Makefile `e2e-test` please

Reply:
Done.